packages/hardhat/contracts/MultiUriMetadata.sol

```diff
@@ -4,13 +4,15 @@ pragma solidity ^0.8.6;
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "./interfaces/IMetadata.sol";
 
+error METADATA_IS_IMMUTABLE();
+
 /** Metadata contract where tokenURI is ipfs://<cid>/<tokenId */
 contract MultiUriMetadata is IMetadata, Ownable {
     event URIChanged(string indexed newURI);
     event MetadataFrozen();
 
     string public baseURI;
-    bool public metadataFrozen;
+    bool metadataFrozen;
```

Comment:
curious why not public

Reply:
My bad I had a function earlier missed to add pushing